src/main/kotlin/io/snyk/plugin/SnykBulkFileListener.kt

```diff
@@ -75,13 +79,20 @@ class SnykBulkFileListener : BulkFileListener {
             // clean IaC cached results if needed
             val currentIacResult = toolWindowPanel.currentIacResult
             if (currentIacResult != null) {
-                val iacRelatedFileChanged = virtualFilesAffected
-                    .filter { iacFileExtensions.contains(it.extension) }
-                    .find { ProjectRootManager.getInstance(project).fileIndex.isInContent(it) }
-                if (iacRelatedFileChanged != null) {
-                    toolWindowPanel.currentIacResult = null
-                    LOG.debug("IaC cached results dropped due to changes in: $iacRelatedFileChanged")
-                    DaemonCodeAnalyzer.getInstance(project).restart()
+                RunUtils.instance.runInBackground(
```

Comment:
don't use Code runner ostside of Code activities

Reply:
Use ProgressManager...

Reply:
Let's see if we need currentIacResult for Annotations and such.